examples/cloud_run_cloudevents/README.md

```diff
@@ -0,0 +1,41 @@
+# Deploying a CloudEvent function to Cloud Run with the Functions Framework
+
+This sample uses the [Cloud Events SDK](https://github.com/cloudevents/sdk-python) to send and receive a CloudEvent on Cloud Run.
+
+## How to run this locally
+
+Build the Docker image:
+
+```commandline
+docker build -t ff_example .
+```
+
+Run the image and bind the correct ports:
+
+```commandline
+docker run --rm -p 8080:8080 -e PORT=8080 ff_example
+```
+
+Send an event to the container:
```

Comment:
How do you run this? What is the name of this new file?

Below is code rather than a command. Have exact instructions on how to run this.

Reply:
Revised readme and moved the code into a file. README now tells user to run the file